packages/expo-dev-menu-interface/expo-module.config.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Do you plan on moving `expo-dev-menu-interface` to the core? Would we benefit of it?